library/sql.inc

```diff
@@ -21,8 +21,8 @@
 */
 
 require_once(dirname(__FILE__) . "/sqlconf.php");
-require_once(dirname(__FILE__) . "/adodb/adodb.inc.php");
-require_once(dirname(__FILE__) . "/adodb/drivers/adodb-mysqli.inc.php");
+require_once("{$GLOBALS['vendor_dir']}/adodb/adodb-php/adodb.inc.php");
+require_once("{$GLOBALS['vendor_dir']}/adodb/adodb-php/drivers/adodb-mysqli.inc.php");
```

Comment:
Just so I am understanding things here, adodb can't use the auto loader because it's using functions (such as NewADOConnection() ) rather than classes. correct?